db/schema.rb

```diff
@@ -330,7 +329,7 @@
     t.jsonb    "preferences",         default: {}
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "roles",               default: [], null: false, array: true
+    t.jsonb    "activity_count",      default: {}
```

Comment:
So it keeps per workflow counts as the content coming through via the events api [here](https://github.com/zooniverse/Zooniverse-Home/blob/773965dd8d775f941dc8a637cce6024f0bbbf55b/app/controllers/api/events_controller.rb#L25) is a hash that looks like this:
```
 activity_count: {"orchid"=>{"count"=>"2", "updated_at"=>"2015-05-07 13:52:04 UTC"}}>
```
Not sure what the best way of storing those per workflow counts are. Do you know of any project's subscription events in zoo home that would have different workflow counts?